packages/harmony/src/components/button/BaseButton/BaseButton.tsx

```diff
@@ -63,7 +63,7 @@ export const BaseButton = forwardRef<HTMLButtonElement, BaseButtonProps>(
       userSelect: 'none',
       whiteSpace: 'nowrap',
       paddingInline: 0,
-      '-webkit-padding': 0,
+      WebkitPadding: 0,
```

Comment:
is this an actual value? just curious ik the browser says it... but what does this even do? i can't find any docs for it

Reply:
agreed, will drop